src/js/common/schemaform/definitions/nonMilitaryJobs.js

```diff
@@ -0,0 +1,24 @@
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+import EmploymentPeriodView from '../../../edu-benefits/components/EmploymentPeriodView';
+
+export const schema = commonDefinitions.nonMilitaryJobs;
```

Comment:
If I use the default export, do I still name the object `uiSchema`? A little confused on how to use default exports from reading the docs.